_posts/2016-08-15-july-meetup-recap.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: post
+title: July Open Gov & Civic Tech Meetup Recap-Hacking the Budget
```

Comment:
Our previous  blog post are formatted the way I have the title now. Following that convention.